### PR TITLE
Added VIA folder under the keymaps folder

### PR DESCRIPTION
**Pull request by rojasa1990**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by rojasa1990** [comment]
Added a VIA folder that contains a keymap.c and rules.mk files to support VIA

File: keyboards/genone/eclipse_65/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```
Comment (drashna) [inline keyboards/genone/eclipse_65/keymaps/via/keymap.c:19]: VIA requires that all of the layers be populated.  

You need to add 2 more layers (for a total of 4), or set the max layers to 2

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/genone/eclipse_65/keymaps/via/keymap.c (hunk 1 above)
Comment (rojasa1990) [reply keyboards/genone/eclipse_65/keymaps/via/keymap.c:19]: Updated the keymap.c file to include 4 layers as requested.

**Comment by rojasa1990** [comment]
Thank you so much for messaging me back. I just made changes to the file to include 4 layers.

________________________________

**Review by drashna — APPROVED** [review]

**Review by rojasa1990 — COMMENTED** [review]
Approved changes made.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks for the contribution!

**Comment by drashna** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Fixing Your Branch](https://docs.qmk.fm/#/newbs_git_resynchronize_a_branch) will walk you through fixing up your `master` branch moving forward. If you need any help with this just ask.

Thanks for contributing!

**Comment by rojasa1990** [comment]
Thanks for the information that you just gave me. I am going to ask some questions below if you don't mind. I am not a programmer and I do not have a lot of experience with code, I am actually a biochemist with an interest in keyboard and computers.

1) So. Instead of doing a pull request from github directly, I can make changes to my files using a command interface like MSYS?
2) I am trying to get VIA support for my keyboard. Do I have to make my own fork/repository for VIA? just like how I did for QMK firmware.
3) I'm trying to get more information for VIA support and I went to thier DOCS and all it says is to make a via keymap directory in my QMK source. I have done this and created a json file which works, I am lost now, any help would be greatly appreciated.

________________________________

**Pull request merged**